src/lib/tecs/TECS.cpp

```diff
@@ -88,12 +88,14 @@ void TECS::update_vehicle_state_estimates(float equivalent_airspeed, const float
 
 	// Update and average speed rate of change if airspeed is being measured
 	if (PX4_ISFINITE(equivalent_airspeed) && airspeed_sensor_enabled()) {
+		_tas_rate_raw = speed_deriv_forward;
 		// Apply some noise filtering
 		_TAS_rate_filter.update(speed_deriv_forward);
-		_speed_derivative = _TAS_rate_filter.getState();
+		_tas_rate_filtered = _TAS_rate_filter.getState();
```

Comment:
I don't think it is useful to keep a copy of the filter's output since you can call `_TAS_rate_filter.getState()` where you need it, without loss of performance.

Reply:
@bresch True, we can get rid of this